### ISSUE TITLE
Source of test set 2D detections

### ISSUE THREAD

**Issue by nazMahmoud** [pangsu0613/CLOCs#48]
Hi @pangsu0613 , what is the source of the 2D detections that was used for CLOCs KITTI test submission [CLOCs_PVCas](http://www.cvlibs.net/datasets/kitti/eval_object_detail.php?&result=51ac3002b28f9f7d2811193799b8e5db8314c0cc) ? is it the pre-trained model on training and validation provided by [cascaded rcnn](https://github.com/zhaoweicai/mscnn/blob/master/examples/kitti_car/run_cascademscnn.m) ?

Thank you for the support.

**Comment by pangsu0613** [comment]
Hello @nazMahmoud , for CLOCs_PVCas, I used the pre-trained model provided by cascade-rcnn, their pre-trained model is trained on training + validation sets (7481 samples).

**Comment by nazMahmoud** [comment]
Thank you!
